src/cjs.js

```diff
@@ -1 +1,4 @@
-module.exports = require('./index').default;
+const fileLoader = require('./index');
+
+module.exports = fileLoader.default;
+module.exports.raw = fileLoader.raw;
```

Comment:
@Kovensky What are potential issue with mutating `loader.default` ? Handling the Module System related stuff in the wrapper would 👍  if possible 🙃